ClientRuntimes/CSharp/ClientRuntime.Azure/Microsoft.Rest.ClientRuntime.Azure/project.lock.json

```diff
@@ -0,0 +1,2071 @@
+{
```

Comment:
It is "locking" the dependencies calculated at that time. dnu build uses what is in the project.lock.json